homework-2/script.js

```diff
@@ -0,0 +1,60 @@
+// Output the numbers from 20 to 30 through the gap using a step of 0.5 (20 20.5 21 21.5….)
+
+function listIntegerFloatNumbers(x, y) {
```

Comment:
bad name - Integers and floats are two different kinds of numerical data. listHalfStepNumbers